kube/resources/volume-configs.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
     airbyte: volume-configs
 spec:
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
```

Comment:
is it a problem that both scheduler and server read from this volume? or is that mitigated by them being on the same node?

Reply:
A built in `ReadWriteMany` volume isn't available on GCP. `ReadWriteOnce` is available but only one access mode can be used for a volume and it can only be on a single node. Pod affinity keep the scheduler and server on the same node.